packages/math-toolbar/src/editor-and-pad.jsx

```diff
@@ -273,6 +273,9 @@ const styles = theme => ({
     },
     '& .mq-overarrow-inner-left': {
       display: 'none !important'
+    },
```

Comment:
I see we're using em here. Do we set the fontSize of the parent element (do we have control on it?)?